### PR TITLE
Add Tamil translation directory and templates

### PR DESCRIPTION
**Pull request by millerdev**
Content is currently Hindi, and needs to be updated.

### CODE REVIEW

**Comment by millerdev** [comment]
Accidentally self-merged before review/approval because this PR is mostly the work of @marissahrrsn. I'll be happy to revert if there is an issue found in review (although I think it should be fine).

File: custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{# NOTE: Django assumes templates are stored with UTF-8 encoding by default. Be sure to save this template with UTF-8 encoding. #}
+
+रिकार्ड के जानकारी के आधार पर आापके केन्द्र में पूर्व माह में VHSND नहीं आयोजित किया गया है। कृपया प्रत्येक माह नियमित रूप से VHSND का आयोजन सुनिश्चित करें।
```
Comment (millerdev) [inline custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt]: Hindi?

File: custom/icds/templates/icds/messaging/indicators/tam/child_vaccinations_complete.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{# NOTE: Django assumes templates are stored with UTF-8 encoding by default. Be sure to save this template with UTF-8 encoding. #}
+
+बाधाई हो ! आपने {{ child_name }} को सभी टीके लगवा दिए हैं |
```
Comment (millerdev) [inline custom/icds/templates/icds/messaging/indicators/tam/child_vaccinations_complete.txt]: Hindi?

File: custom/icds/templates/icds/messaging/indicators/tam/dpt3_and_measles_due.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{# NOTE: Django assumes templates are stored with UTF-8 encoding by default. Be sure to save this template with UTF-8 encoding. #}
+
+{{ child_name }} को इस महीने डी. पी. टी.-३ और खसरे   का टीका लगाना हैं पर अभी तक उसे यह टीका नहीं लगे हैं |
```
Comment (millerdev) [inline custom/icds/templates/icds/messaging/indicators/tam/dpt3_and_measles_due.txt]: Hindi?

**Review by millerdev — COMMENTED** [review]
@marissahrrsn looks like a few templates were missed (although I can't read Tamil or Hindi, so I might be wrong). Was that intentional? Should the Hindi templates be removed?

File: custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt (hunk 1 above)
Comment (marissahrrsn) [reply custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt]: Correc, this is Hindi. I replaced all of the Hindi in the templates with the Tamil translations.

File: custom/icds/templates/icds/messaging/indicators/tam/child_vaccinations_complete.txt (hunk 2 above)
Comment (marissahrrsn) [reply custom/icds/templates/icds/messaging/indicators/tam/child_vaccinations_complete.txt]: Yep, that is Hindi. I removed it and replaced it with Tamil.

File: custom/icds/templates/icds/messaging/indicators/tam/dpt3_and_measles_due.txt (hunk 3 above)
Comment (marissahrrsn) [reply custom/icds/templates/icds/messaging/indicators/tam/dpt3_and_measles_due.txt]: Yes, Hindi. This indicator was never released, so I didn't add Tamil translations.

File: custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt (hunk 1 above)
Comment (millerdev) [reply custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt]: @marissahrrsn I must have missed the PR with the updated translation for this one. Can you link me to it?

File: custom/icds/templates/icds/messaging/indicators/tam/child_vaccinations_complete.txt (hunk 2 above)
Comment (millerdev) [reply custom/icds/templates/icds/messaging/indicators/tam/child_vaccinations_complete.txt]: @marissahrrsn I must have missed the PR with the updated translation for this one. Can you link me to it?

Comment (millerdev) [reply custom/icds/templates/icds/messaging/indicators/tam/child_vaccinations_complete.txt]: Ignore previous comment. Posted in wrong place.

File: custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt (hunk 1 above)
Comment (millerdev) [reply custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt]: I was reading too quickly. Please ignore my previous comment. Sorry for the noise.

**Pull request merged**